pkg/tests/cross-tests/diff_check.go

```diff
@@ -58,8 +58,8 @@ func runDiffCheck(t T, tc diffTestCase) {
 	tfwd := t.TempDir()
 
 	tfd := newTfDriver(t, tfwd, providerShortName, rtype, tc.Resource)
-	_ = tfd.writePlanApply(t, tc.Resource.Schema, rtype, "example", tc.Config1)
-	tfDiffPlan := tfd.writePlanApply(t, tc.Resource.Schema, rtype, "example", tc.Config2)
+	_ = tfd.writePlanApply(t, tc.Resource.SchemaMap(), rtype, "example", tc.Config1)
```

Comment:
Resource may specify Schema or SchemaFunc, and SchemaMap() normalizes to access either, this is the way.